remote-trait-object/src/service.rs

```diff
@@ -14,8 +14,8 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-pub trait ServiceHandler: Send {
-    fn call(&self, input: String) -> String {
+pub trait ServiceDispatch: Send {
+    fn dispatch(&self, input: String) -> String {
```

Comment:
dispatch is a process to find a method to call. What we want to do is finding the method and calling it.
I don't know a better name though. Maybe `dispatch_and_call`?